var/spack/repos/builtin/packages/elpa/package.py

```diff
@@ -48,11 +51,18 @@ class Elpa(AutotoolsPackage, CudaPackage, ROCmPackage):
 
     variant("openmp", default=True, description="Activates OpenMP support")
     variant("mpi", default=True, description="Activates MPI support")
-    variant("gpu_streams", default=True, description="Activates GPU streams support")
+
+    with when("@2021.11.01:"):
```

Comment:
```suggestion
    with when("@2021.11.001:"):
```
? I noticed that in https://github.com/spack/spack/pull/44749#issuecomment-2180065108 the two versions were different (one was missing a `0` in the bigfix version).

Reply:
oooh! Yes! That was indeed the issue! :)
ok thanks, sorry for the mess!